### DISCUSSION TITLE
What does the `-q` switch do?

### DISCUSSION THREAD

**Discussion by ghost** [lcn2/calc#123] [Q&A] [answered]
I have seen `calc -q -f` is used to execute `.calc` scripts. But I don't know exactly what the `-q` switch does. `-f` seems to be reading from file and I'm pretty sure about that. But I have no idea about `-q`. I tried to omit `-q` and the script still runs as expected.

**Comment by lcn2** [comment, marked as answer]
For the record, `calc -q`, as documented in the calc man page:

```
       -q     Disable the reading of the startup scripts.

              This allows the script to run independently of startup scripts
              such those managed by the $CALCRC environment variable.  For
              example, this will disable the use of the common calcrc file
              (usually ~/.calcrc).
```
